src/components/model.jsx

```diff
@@ -249,4 +265,4 @@ Model.defaultProps = {
   model: null,
 };
 
-export default withApp(withModel(Model, async app => app.getTablesAndKeys({}, {}, 0, true, false)));
+export default withApp(withModel(Model, async app => app.getTablesAndKeys({}, {}, 0, true, false), true));
```

Comment:
Can we use an object into `withModel` instead so it's clearer what the parameters do?

Reply:
Yes, good point!